packages/dx-react-grid/src/components/group-panel-layout.test.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { format } from 'util';
+
+import { GroupPanelLayout } from './group-panel-layout';
+
+// eslint-disable-next-line react/prop-types
+const groupPanelCellTemplate = () => (
+  <div
+    className="cell"
+  />
+);
+
+describe('GroupPanelLayout', () => {
+  /* eslint-disable no-console */
+  const consoleWarn = console.warn;
+  const consoleError = console.error;
+
+  const logToError = (...args) => {
+    throw new Error(format(...args).replace(/^Error: (?:Warning: )?/, ''));
```

Comment:
I've created the related [PR](https://github.com/DevExpress/devextreme-reactive/pull/140)